src/components/App.tsx

```diff
@@ -0,0 +1,158 @@
+import { useState, useEffect, useRef } from "react";
+import React from "react";
+import { ItoDo } from "../components/types/data";
+import { TodoList } from "../components/TodoList";
+import { title } from "process";
+import ReactDOM from "react-dom";
+
+const App: React.FC = () => {
+  const [value, setValue] = useState("");
+  const [search, searchValue] = useState("");
+  let [todos, setTodos] = useState<ItoDo[]>([]);
+  let [condition, setCondition] = useState("All");
+
+  const toDoViewing = () => {
+    setCondition((condition = "Todo"));
```

Comment:
You shouldn't change localState value like that (condition = 'Todo'). Instead of this use
setCondition('Todo')

Reply:
done